src/types.rs

```diff
@@ -64,6 +64,15 @@ pub struct epoll_event {
     _priv: (),
 }
 
+/// Opaque types, you should use [`futex_waitv`] instead.
+///
+/// [`futex_waitv`]: https://docs.rs/linux-raw-sys/latest/linux_raw_sys/general/struct.futex_waitv.html
+#[repr(C)]
+#[allow(non_camel_case_types)]
+pub struct futex_waitv {
```

Comment:
For types missing in libc, I would prefer to export from sys mod, see `OpenHow` and `Timespec`.

Reply:
> For types missing in libc, I would prefer to export from sys mod, see `OpenHow` and `Timespec`.

Do you mean only types? What about constants of `futex_flags`? Currently `FUTEX2_SIZE_U32` must be specified for these operations, or they will always fail.

Reply:
I think just type is enough, user defining FUTEX2_SIZE_U32 seems risk-free. In the long term we will use libc's definitions.